agent/bench-scripts/postprocess/linpack-postprocess-sample

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use lib $ENV{'pbench_lib_dir'};
+use lib $ENV{'pbench_bspp_dir'};
+no lib ".";
+use GenData qw(gen_data);
+use BenchPostprocess qw(
+    create_uid calc_aggregate_metrics convert_samples_hash_to_array
+    create_graph_hash get_label);
+use File::Basename;
+use Data::Dumper;
+use JSON;
+
+my $script = basename($0);
+my $dir = $ARGV[0];
+my $tool_label_pattern = $ARGV[1];
+my $tool_group = $ARGV[2];
+
+my $benchmark_name = 'linpack';
+my $primary_metric = 'gflops';
+
+my %workload;
+my %parameters;
+my @benchmark;
+my %benchmark_params;
+my %throughput;
+my @gflops;
+
+my @client_hostname_dirs;
+if (opendir(my $clients_dh, "$dir/clients")) {
+    @client_hostname_dirs = grep(!/^\./, (sort readdir($clients_dh)));
+    closedir($clients_dh);
+} else {
+    printf STDERR "ERROR: Could not open clients directory!\n";
+    exit 1;
+}
+
+my %client_data;
+foreach my $client_hostname_dir (@client_hostname_dirs) {
+    $client_data{$client_hostname_dir} = {};
+
+    my @client_data_files;
+    if (opendir(my $clients_dh, "$dir/clients/$client_hostname_dir")) {
+        @client_data_files = grep(/^linpack\..+$/, (sort readdir($clients_dh)));
+        closedir($clients_dh);
+    } else {
+        printf STDERR "ERROR: Could not open client %s results directory!\n", $client_hostname_dir;
+        exit 2;
+    }
+
+    foreach my $client_data_file (@client_data_files) {
+        if ($client_data_file =~ /^linpack\.(out|meta)$/) {
+            if (open(CLIENT_OUTPUT, "$dir/clients/$client_hostname_dir/$client_data_file")) {
+                while (<CLIENT_OUTPUT>) {
+                    if ($client_data_file =~ /meta$/) {
+                        if ($_ =~ /^(binary|use_omp|threads|kmp_affinity|numactl_cmd|problem_sizes|leading_dimensions|alignment_values)/) {
```

Comment:
This is basically a rename of `linpack-postprocess` to `linpack-postprocess-sample` with l.59 being the only difference, correct?

Reply:
I went through the individual commits and figured out that this is indeed the case.